tests/integration.rs

```diff
@@ -48,6 +48,32 @@ fn netflix() {
     );
 }
 
+#[test]
+fn sanofi() {
```

Comment:
nit: I think it would be clearer to name this test something related to the RSA sig params. IMO it's less interesting to know where the certificate originated vs what is special about it.